Fruitties/iosApp/iosApp/IOSViewModelStoreOwner.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+import shared
+
+/// A ViewModelStoreOwner specifically for iOS.
+/// This is used with from iOS with Kotlin Multiplatform (KMP).
+class IOSViewModelStoreOwner: ObservableObject {
```

Comment:
Should `IOSViewModelStoreOwner` implement `ViewModelStoreOwner`?

Reply:
I had to add an interface into the shared code, otherwise because it wasn't used anywhere in the common code, it wasn't picked into the Swift Framework.